packages/dd-trace/src/appsec/iast/analyzers/sql-injection-analyzer.js

```diff
@@ -6,7 +6,7 @@ class SqlInjectionAnalyzer extends InjectionAnalyzer {
     super('SQL_INJECTION')
     this.addSub('apm:mysql:query:start', ({ sql }) => this.analyze(sql))
     this.addSub('apm:mysql2:query:start', ({ sql }) => this.analyze(sql))
-    this.addSub('apm:pg:query:start', ({ statement }) => this.analyze(statement))
+    this.addSub('apm:pg:query:start', ({ query }) => this.analyze(query.text))
```

Comment:
I one day it changes again and `query` is `undefined`, `query.text` will throw an Error.

Reply:
It's actually checked in the [instrumentation](https://github.com/DataDog/dd-trace-js/blob/master/packages/datadog-instrumentations/src/pg.js#L36)

IMO, publishing a message in a query:start channel without the query object has no sense, but we can re-check it here for redundancy.